Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -153,7 +153,7 @@ public override void Execute(CancellationToken cancellationToken)
                         }
                         else
                         {
-                            InstallFile(sourceFile, cancellationToken, i == _versionContentSummary.Files.Length - 1);
+                            InstallFile(sourceFile, cancellationToken);
```

Comment:
Lack of `_localMetaData.SaveData()` somewhere here.

Reply:
done